diff -Naur fuse-utils-1.0.0/Makefile.in fuse-utils-1.0.0-mod/Makefile.in
--- fuse-utils-1.0.0/Makefile.in	2010-12-16 23:48:54.000000000 +0200
+++ fuse-utils-1.0.0-mod/Makefile.in	2010-12-17 23:37:21.000000000 +0200
@@ -348,7 +348,7 @@
 rzxcheck_SOURCES = rzxcheck.c keys.c utils.c
 rzxcheck_LDADD = @LIBSPEC_LIBS@ compat/@COMPAT_OSNAME@/libcompatos.a
 rzxdump_SOURCES = rzxdump.c utils.c
-rzxdump_LDADD = @LIBSPEC_LIBS@ compat/@COMPAT_OSNAME@/libcompatos.a
+rzxdump_LDADD = @LIBSPEC_LIBS@ compat/@COMPAT_OSNAME@/libcompatos.a  -lgcrypt
 rzxtool_SOURCES = rzxtool.c utils.c
 rzxtool_LDADD = @LIBSPEC_LIBS@ compat/@COMPAT_OSNAME@/libcompatos.a
 scl2trd_SOURCES = scl2trd.c
